src/lib.rs

```diff
@@ -1,11 +1,59 @@
-pub mod analyzer;
-pub mod detection;
+//! Charabia library tokenize a text detecting the Script/Language, segmenting, normalizing, and classifying it.
+//!
+//! Examples
+//! --------
```

Comment:
Are you sure it is creating a header-1 (h1)?

Reply:
The rendered version is:
<img width="995" alt="Capture d’écran 2022-05-03 à 14 27 14" src="https://user-images.githubusercontent.com/6482087/166452647-d19e04eb-dd74-4bfe-8e92-062b5f79d62b.png">

and I find it clear, what do you think?